baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -41,6 +41,7 @@ The definition of `OWS` is taken from [[RFC7230]], Section 3.2.3: https://tools.
 ### list
 List of key-value pairs with optional properties attached.
 It can not be guaranteed that keys are unique.
+The order duplicate entries SHOULD be preserved when mutating the list.
```

Comment:
A normative requirement here directly contradicts the addition you made below where producer/consumer can define their own precedence when deduplicating keys.

Reply:
I will remove that.